drivers/staging/vc04_services/interface/vchiq_arm/vchiq_arm.c

```diff
@@ -3656,6 +3657,9 @@ static int vchiq_probe(struct platform_device *pdev)
 
 	bcm2835_camera = vchiq_register_child(pdev, "bcm2835-camera");
 	bcm2835_audio = vchiq_register_child(pdev, "bcm2835_audio");
+	bcm2835_codec = vchiq_register_child(pdev, "bcm2835-codec");
+	if (IS_ERR(bcm2835_codec))
```

Comment:
Why the inconsistent error handling? If you change the others (some of which were added in this patch set) to match in a later commit, is it not better to keep them consistent here then change them all?

Reply:
Stefan's patchset has not been merged upstream as they wanted changes. Shall I make a fixup to his patch first include the error handling, and then add the new ones to match?

Reply:
OK, will do.